src/screens/FormScreen.js

```diff
@@ -95,6 +97,7 @@ export const FormScreen = () => {
                   />
                   <BoldText>E-Mail</BoldText>
                   <TextInput
+                    accessibilityLabel={`(Geben Sie Ihren E-Mail addresse ein) ${email}`}
```

Comment:
Geben Sie Ihre E-Mail Adresse ein

Reply:
E-Mail-Adresse

Reply:
Could we have those texts in a text file for those locale strings? maybe a separate text file for a11y makes sense?